### PR TITLE
Update slightly complex Chromium data for webextensions.api

### PR DESCRIPTION
**Pull request by queengooborg**
This PR updates and corrects version values for Chromium (Chrome, Opera, Samsung Internet, WebView Android) for the `api` member of the `webextensions` Web Extensions interface. This sets the feature(s) to a version range based upon the date that the feature was added to BCD with the intent of replacing `true` values with ranged values to eliminate `true` values from BCD.

Commit/PR Adding the Feature: #166

This is a cherry-pick of the files containing more complicated changes that were originally in #23450.  While most of the features are set to ≤58, some were set to different version numbers to satisfy the consistency linter, or there was other readily available information already in BCD to confirm the earlier version number.

### CODE REVIEW

File: webextensions/api/tabs.json [hunk 1]
```diff
@@ -2362,7 +2356,10 @@
                 "version_added": "54",
                 "notes": "Available for use in Manifest V2 only."
               },
-              "opera": "mirror",
+              "opera": {
```
Comment (rebloor) [inline webextensions/api/tabs.json:2359]: @queengooborg Where does this come from?

File: webextensions/api/windows.json [hunk 2]
```diff
@@ -672,10 +672,12 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/windows/create",
             "support": {
               "chrome": {
-                "version_added": true,
+                "version_added": "≤44",
                 "notes": "<code>top</code>, <code>bottom</code>, <code>left</code>, <code>right</code> bounds must be at least 50% within visible screen space if specified, otherwise an error is thrown."
               },
-              "edge": "mirror",
+              "edge": {
```
Comment (rebloor) [inline webextensions/api/windows.json:678]: @queengooborg Where did this come from?

File: webextensions/api/windows.json [hunk 3]
```diff
@@ -1099,7 +1101,9 @@
               "firefox_android": {
                 "version_added": false
               },
-              "opera": "mirror",
+              "opera": {
```
Comment (rebloor) [inline webextensions/api/windows.json:1104]: @queengooborg source?

File: webextensions/api/tabs.json (hunk 1 above)
Comment (queengooborg) [reply webextensions/api/tabs.json:2359]: This came from mirroring from Chrome, just with dropping the range denominator to satisfy the consistency linter!

File: webextensions/api/windows.json (hunk 2 above)
Comment (queengooborg) [reply webextensions/api/windows.json:678]: This came from the version numbers present in downstream features!

**Review by rebloor — APPROVED** [review]

**Pull request merged**